packages/okta-adapter/src/user_config.ts

```diff
@@ -124,3 +128,31 @@ export const configType = definitions.createUserConfigType({
   omitElemID: false,
   pathsToOmitFromDefaultConfig: ['fetch.enableMissingReferences', 'fetch.getUsersStrategy'],
 })
+
+/*
+ * Temporary config suggestion to migrate existing configs to exclude UserRoles type
+ */
+export const getExcludeUserRolesConfigSuggestion = (
+  userConfig: Readonly<InstanceElement> | undefined,
+): definitions.ConfigChangeSuggestion | undefined => {
+  const typesToExclude = userConfig?.value?.fetch?.exclude
+  const typesToInclude = userConfig?.value?.fetch?.include
+  if (!Array.isArray(typesToExclude) || !Array.isArray(typesToInclude)) {
+    log.error(
+      'failed creating config suggestion to exclude UserRoles type, expected fetch.exclude and fetch.include to be an array, but instead got %s',
+      safeJsonStringify({ exclude: typesToExclude, include: typesToInclude }),
+    )
+    return undefined
+  }
+  const isUserRolesExcluded = typesToExclude.find(fetchEnty => fetchEnty?.type === USER_ROLES_TYPE_NAME)
```

Comment:
typo nit:
```suggestion
const isUserRolesExcluded = typesToExclude.find(fetchEntry => fetchEntry?.type === USER_ROLES_TYPE_NAME)
```
(And in all other places)